servicebus/templates/servicebus.yaml

```diff
@@ -0,0 +1,48 @@
+{{- $release := .Release | default dict -}}
+{{- $globals := .Values.global | default dict -}}
+{{- $tags := $globals.tags | default dict -}}
+{{- $base := . -}}
+{{- if .Values.setup -}}
+  {{- if .Values.setup.queues -}}
+    {{- range $queue := .Values.setup.queues }}
+---
+apiVersion: servicebus.azure.com/v1beta20210101preview
```

Comment:
would be good to split them into files and add add optional servicebus namespace.